bin/ilocate

```diff
@@ -20,21 +18,29 @@ then
 	echo "Options are :"
 	echo "-0  Separate output via NULL characters instead of newlines (like find -print0)" 
 	echo "-t  Also show objects in trash"
+	echo "-i  Use case-insensitive search"
         echo ""
         ienv -h | tail -n1 | sed s/ienv/ilocate/
 	exit 0
 fi
 
+case_insensitive=0
+
 iQuestQry ()  # $1 == query_column, $2 == inputstr
 {
-		sql_output=`iquest --no-page "%s/%s" "select COLL_NAME,DATA_NAME WHERE $1 $operand '$2'" 2> /dev/null`
-		if [ $? -eq 0 ]
-		then
-			echo "$sql_output" | grep -v "CAT_NO_ROWS_FOUND"
-			return 0
-		else
-			return 1
-		fi
+        if [ $case_insensitive -eq 0 ]; then
+		sql_output=`iquest --no-page "%s/%s" "select COLL_NAME, DATA_NAME WHERE $1 $operand '$2'" 2> /dev/null`
+        else
+		sql_output=`iquest uppercase --no-page "%s/%s" "select COLL_NAME, DATA_NAME WHERE $1 $operand '${2^^}'" 2> /dev/null`
```

Comment:
this requires bash 4+?   please confirm that's shipped with our supported OS/versions.

Reply:
replaced the solution with a bash 3 impl. the `awk` impl should be safe.

thoughts?

Reply:
seems good.

we have a test?

Reply:
i didn't see any tests for it. i'll look again.

Reply:
Added test.